diff --git a/fesm2015/core.mjs b/fesm2015/core.mjs
index 7f4a5b2b766ccaae6c0eb3eb87f22e606c05de80..0c5e1f6a37bf805fff3bbfdfcb59e0ab8b4bf38c 100755
--- a/fesm2015/core.mjs
+++ b/fesm2015/core.mjs
@@ -5,7 +5,7 @@
  */
 
 import { Subject, Subscription, Observable, merge as merge$1 } from 'rxjs';
-import { share } from 'rxjs/operators';
+import { share } from 'rxjs/operators/index.js';
 
 /**
  * @license
diff --git a/fesm2020/core.mjs b/fesm2020/core.mjs
index aa71aed4a4cf8df1193204038f1d34f30b80639e..6e03c107aa858b590757cae1b70dc2b14cea083b 100755
--- a/fesm2020/core.mjs
+++ b/fesm2020/core.mjs
@@ -5,7 +5,7 @@
  */
 
 import { Subject, Subscription, Observable, merge as merge$1 } from 'rxjs';
-import { share } from 'rxjs/operators';
+import { share } from 'rxjs/operators/index.js';
 
 /**
  * @license
